src/simulations/modeling/adapter.py

```diff
@@ -487,6 +505,27 @@ def bounds(measurement, uncertainty):
                 }
             )
 
+    for measure in proteomics:
```

Comment:
We should probably verify that we have an ecModel here too, like in the flexibilization step above (I would only add the warning here; since the above logic is dependent on whether growth rate was applied).

Reply:
Fully agree, the warning makes more sense here. Done in commit [`51822fb`](https://github.com/DD-DeCaF/simulations/pull/147/commits/51822fb586bf93fdad06494f7d8387a193855aeb)